### PR TITLE
style: adjust docs sidebar chevron icon

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksadjust-chevron-direction-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1204807665183200/1205002759200577/f) 🎟️
- [Figma file](https://www.figma.com/file/G6RNneOMHyRgcFx98VZUpp/IA-Improvements?type=design&node-id=406-54557&mode=design&t=rG3DKEUgroeRBpsY-0) 🎨 

## 🗒️ What

Updates the nested sidebar chevron icon to point down / up to align with the dropdown disclosure styles. 

## 📸 Design Screenshots
Screenshot 2023-07-11 at 2 12 47 PM

## 🧪 Testing

- [Visit a docs page ](https://dev-portal-git-ksadjust-chevron-direction-hashicorp.vercel.app/consul/docs)with nested sidebar items
- Validate that the chevron points down when closed and up when opened.

### CODE REVIEW

File: src/components/sidebar/components/sidebar-nav-menu-item/sidebar-nav-menu-item.module.css [hunk 1]
```diff
@@ -31,12 +31,19 @@
 		background-color: var(--token-color-palette-neutral-200);
 	}
 
+	& svg {
+		/* Only enable animation if query is supported and value is no-preference */
+		@media (prefers-reduced-motion: no-preference) {
```
Comment (kendallstrautman) [inline src/components/sidebar/components/sidebar-nav-menu-item/sidebar-nav-menu-item.module.css:36]: Added a transition ✨

File: src/components/sidebar/components/sidebar-nav-menu-item/sidebar-nav-menu-item.module.css [hunk 2]
```diff
@@ -31,12 +31,19 @@
...
+		}
+	}
+
 	/* AKA open */
 	&[aria-expanded='true'] {
 		margin-bottom: 0;
 
 		& svg {
-			transform: rotate(90deg);
+			transform: rotate(-180deg);
```
Comment (kendallstrautman) [inline src/components/sidebar/components/sidebar-nav-menu-item/sidebar-nav-menu-item.module.css:46]: Tried to match the direction of the dropdown disclosure.

**Review by BrandonRomano — APPROVED** [review]
Looks good!

**Review by braydenlove — APPROVED** [review]
Nicely done, thanks!

**Pull request merged**